features/dapp_develop/cli.go

```diff
@@ -93,17 +93,13 @@ func getEventsFromCliTool(ledgerFrom uint32, deployedContractId string, size uin
 
 	status, stdOutLines, err := e2e.RunCommand(envCmd, e2eConfig)
 	var jsonEvents []map[string]interface{}
-	var stdOutLinesTrimmed []string
 
 	if status != 0 || err != nil {
 		return jsonEvents, fmt.Errorf("soroban cli get events had error %v, %v", status, err)
 	}
 
-	// put commas between any json event objects if more than one found
-	stdOutEventsValidJson := strings.ReplaceAll(strings.Join(stdOutLinesTrimmed, "\n"), `\n}\n{\n`, `\n}\n,\n{\n`)
-	// wrap the json objects in json array brackets
-	stdOutEventsValidJson = "[" + stdOutEventsValidJson + "]"
-
```

Comment:
I think the code being deleted here is trying to deal with the JSON returned being a stream of JSON objects, rather than a JSON array. It's unclear from the PR description/title what is being fixed. Can you elaborate?

Reply:
The CLI should output a list of events in JSON. the consumer should be able use it as input that is valid JSON.

Reply:
A stream of JSON is valid though. Why are we changing the output of the events endpoint?

Reply:
It seemed that theses tests weren't viewing it as stream and rather converted the one input into one valid JSON string. But I removed it. We can deal with that in a separate PR. It would be better if the options were `--json` and `--json-stream`. As `json` doesn't imply a stream.